releasenotes/notes/add-dd_call_basic_config-env-15570281d176f438.yaml

```
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add ``DD_CALL_BASIC_CONFIG={true,false}`` environment variable to control whether ``ddtrace`` calls ``logging.basicConfig``.
+    By default when using ``ddtrace-run`` or running in debug mode, we always call ``logging.basicConfig`` to ensure there is always a
```

```suggestion
    By default when using ``ddtrace-run`` or running in debug mode ``logging.basicConfig`` is called to ensure there is always a
```

I always find the "we/us" language weird when referring to the library 😆